test/known_issues/test-vm-accessor_properties.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+// Ref: https://github.com/nodejs/node/issues/2734
+
+require('../common');
+const vm = require('vm');
+const assert = require('assert');
+
+
+const sandbox = Object.create(null, {
+  prop: {
+    get: function() { return 'foo';}
+  }
```

Comment:
Object.create/defineProperty(ies) uses this syntax for getters and setters in the accessor properties, which is what we want to fix with new 5.5. V8 (IMHO).
`Defining a getter on existing objects using defineProperty` in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get 

Trying understand why the change would be better:)